packages/date-helpers/src/index.ts

```diff
@@ -34,6 +34,18 @@ export {
  */
 export const currentDate = (): string => new Date().toISOString().slice(0, 10);
 
+/**
+ * Attempts to get the users local timezone
+ */
+export const getLocalTimezone = (): string | undefined => {
+  let localTz;
+  if (window.Intl && typeof window.Intl === "object") {
+    localTz = Intl.DateTimeFormat().resolvedOptions().timeZone;
+  }
+
+  return localTz;
+};
+
 /**
  * Format a date in given format and locale, optionally in a different time zone
```

Comment:
Could you add a comment to make it clearer what the default is when no zone is provided?  Is the default always the local time zone or is it more nunanced than that?  I think it is, given the changes you made to parseDateTimeUtc.

Reply:
It is very slightly more nuanced. I tried to explain it in [improve documentation for timezone behaviour](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12469/commits/50258bfaa9ba7aa710788f57947aebde7540ca3d)

Does that make sense? :sweat_smile:

Reply:
That's better, yeah.  I wish the function didn't have a "it depends" behaviour but it is what is is.  :laughing:

Reply:
haha yeah. We could if instead of dates we just used the timestamp though!